app/views/users/_metadata.html.erb

```diff
@@ -1,34 +1,78 @@
-<% if @user.employment_title.present? || (@user.looking_for_work_publicly == true && @user.looking_for_work == true) || @user.education.present? %>
-  <div class="profile-header__bottom fs-base">
-    <% if @user.education.present? %>
-      <div class="crayons-definition">
-        <strong class="crayons-definition__title">Education</strong>
-        <p class="crayons-definition__value"><%= @user.education %></p>
-      </div>
-    <% end %>
+<% if FeatureFlag.enabled?(:profile_admin) %>
```

Comment:
This should be safe to deploy to DEV because all of the UI changes are behind feature flags.